include/SH3/graphics/quad.hpp

```diff
@@ -0,0 +1,45 @@
+/** @file
+ *
+ *  Class to render a fullscreen 2D quad to the screen.
+ *
+ *  @copyright 2017 Palm Studios
+ *
+ *  @date 11-2-2017
+ *
+ *  @note To render a quad, we need to split the screen into two right-angled triangles, meaning we need 6 vertices for one quad.
+ *
+ *  @author Jesse Buhagiar
+ */
+#ifndef QUAD_HPP_INCLUDED
+#define QUAD_HPP_INCLUDED
+
+#include "GL/glew.h"
+#include "GL/gl.h"
+#include "glm/glm.hpp"
+
+#include <vector>
+
+namespace sh3_graphics
+{
+    struct quad final
+    {
+    public:
+        /**
+         *  Class constructor. Takes
+         */
+        quad(const std::vector<glm::vec3>& verts);
```

Comment:
I think it's superfluous unless we're doing some math. All we need is 3 points, the type `vertex3f` makes more sense imho.